src/dawn/Compiler/Options.h

```diff
@@ -25,6 +25,7 @@ struct Options {
 #define OPT(TYPE, NAME, DEFAULT_VALUE, OPTION, OPTION_SHORT, HELP, VALUE_NAME, HAS_VALUE, F_GROUP) \
   TYPE NAME = DEFAULT_VALUE;
 #include "dawn/Compiler/Options.inc"
+#include "dawn/Optimizer/OptimizerOptions.inc"
```

Comment:
Isn't that more error prone, now you have to do it in multiple places. Why not include it in the Options.inc but in the end if you want it afterwards?

Reply:
So, I wasn't sure about this. It seems reasonable to separate the 2 options subsets, as we may need only one, as in https://github.com/MeteoSwiss-APN/dawn/blob/c4a639d808cb6c1253ee8b1140fbfa96a2f867e5/src/dawn/Compiler/DawnCompiler.cpp#L123 but it is also true that optimizer's options are also compiler's options. Since it's no harm to include more options than necessary, I will include again optimizer's opts into compiler's opts.